<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/V0142_R1_trim_paired.error_rates.pdf">errors</a></td><td>V0142_R1_trim_paired</td><td align="right">1,043,517</td><td align="right">151,644,038</td><td align="right">100.0%</td><td align="right">145.3&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">89.8%</td></tr><tr ><td><a href="calibration/V0142_R2_trim_paired.error_rates.pdf">errors</a></td><td>V0142_R2_trim_paired</td><td align="right">1,035,282</td><td align="right">150,627,589</td><td align="right">99.2%</td><td align="right">145.5&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">88.6%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>2,078,799</b></td><td align="right"><b>302,271,627</b></td><td align="right">99.6%</td><td align="right">145.4&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">89.2%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/CU458896.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>CU458896</td><td align="right">5,067,172</td><td align="center">59.8</td><td align="center">1.7</td><td align="center">100.0%</td><td>Mycobacterium abscessus ATCC 19977 chromosome, complete sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/CU458745.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>CU458745</td><td align="right">23,319</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Mycobacterium abscessus ATCC 19977 plasmid pMAB23, complete</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>5,090,491</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">22302</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">327</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.020</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>CU458896</td><td align="center">0.85019</td></tr><tr >
<td>CU458745</td><td align="center">NA</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.5.1</td></tr>
<tr><td>R</td><td>4.2.3</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>13:00:03&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:00:42&nbsp;07&nbsp;Nov&nbsp;2023</td><td>39&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>13:00:42&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:08:48&nbsp;07&nbsp;Nov&nbsp;2023</td><td>8&nbsp;minutes&nbsp;6&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>13:08:48&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:09:24&nbsp;07&nbsp;Nov&nbsp;2023</td><td>36&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>13:09:24&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:11:01&nbsp;07&nbsp;Nov&nbsp;2023</td><td>1&nbsp;minute&nbsp;37&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>13:11:01&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:17:54&nbsp;07&nbsp;Nov&nbsp;2023</td><td>6&nbsp;minutes&nbsp;53&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>13:17:54&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:19:46&nbsp;07&nbsp;Nov&nbsp;2023</td><td>1&nbsp;minute&nbsp;52&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>13:19:46&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:20:48&nbsp;07&nbsp;Nov&nbsp;2023</td><td>1&nbsp;minute&nbsp;2&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>13:20:48&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:22:25&nbsp;07&nbsp;Nov&nbsp;2023</td><td>1&nbsp;minute&nbsp;37&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>13:22:25&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:23:10&nbsp;07&nbsp;Nov&nbsp;2023</td><td>45&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>13:23:10&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:23:12&nbsp;07&nbsp;Nov&nbsp;2023</td><td>2&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>13:23:12&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:30:06&nbsp;07&nbsp;Nov&nbsp;2023</td><td>6&nbsp;minutes&nbsp;54&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>13:30:06&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:30:15&nbsp;07&nbsp;Nov&nbsp;2023</td><td>9&nbsp;seconds</td></tr>
<tr><td>Output</td><td>13:30:15&nbsp;07&nbsp;Nov&nbsp;2023</td><td>14:43:36&nbsp;07&nbsp;Nov&nbsp;2023</td><td>1&nbsp;hour&nbsp;13&nbsp;minutes&nbsp;21&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>13:30:15&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:39:06&nbsp;07&nbsp;Nov&nbsp;2023</td><td>8&nbsp;minutes&nbsp;51&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>13:39:06&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13:39:19&nbsp;07&nbsp;Nov&nbsp;2023</td><td>13&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>1&nbsp;hour&nbsp;52&nbsp;minutes&nbsp;37&nbsp;seconds</b></td>
</tr>
</table></body></html>